src/backend_skia_bindings/image.rs

```diff
@@ -21,12 +21,13 @@ pub fn draw_raster(
 
     let mut image = {
         let (w, h) = img.size.dimensions();
-        let surface = usvg::try_opt_warn_or!(
-            skia::Surface::new_raster(&skia::ImageInfo::new_unknown(Some(skia::ISize::new(w as i32, h as i32))), None, None), (),
+        let image_info = skia::ImageInfo::new(skia::ISize::new(w as i32, h as i32),skia::ColorType::n32(), skia::AlphaType::Unpremul, None);
```

Comment:
same as this - https://github.com/RazrFalcon/resvg/blob/b70a71293d86e4bd058970600668e4c9217810b5/resvg-skia/skia-bindings/skia_capi.cpp#L67